helpers/cli/cli.js

```diff
@@ -7,21 +7,53 @@ const path = require('path');
 const spawn = require('cross-spawn');
 const inquirer = require('inquirer');
 const cwd = process.cwd();
-//const {pagesDirPath, reframeConfigPath, appDirPath} = find_files(cwd);
-//const reframeConfig = reframeConfigPath && require(reframeConfigPath);
-//const {processReframeConfig} = require('@reframe/utils/processReframeConfig');
 const {questions} = require('./questions');
 let argValue;
 
-//processReframeConfig(reframeConfig);
-//console.log(reframeConfig._processed);
-
 process.on('unhandledRejection', err => {
     console.log(err);
     process.exit(1);
 });
 
-program
+let appExists = fs.existsSync(path.resolve(cwd, 'reframe.config.js'));
```

Comment:
We could remove the `assert_usage` in `find_files` so that we can check if `find_files` found a `reframeConfig` or a `pages/` to determine whether a project has been found or not

Reply:
So how do you want me to implement that?  Right now, at the entry point, I need a check to see if that file exists to know which CLI commands to offer.  Are you wanting me to remove all of the assert_usage stuff in find_files then change it to `if (find_files(cwd)`?  Or are you saying run find_files at the beginning then say `if (reframeConfigPath)`?

Reply:
Yes something like that.
I'd say that no project has been found when both no `reframeConfigPath` and no `pagesDir` has been found.

Reply:
Ok, let me play around with it and see what I can come up with.

Reply:
Ok cool